views/js/pciCreator/ims/graphFunctionInteraction/runtime/graphFunctionInteraction.js

```diff
@@ -181,9 +181,12 @@ define([
          * @param {Object} [state] - the json serialized state object, returned by previous call to getStatus(), use to initialize an
          */
         getInstance : function getInstance(dom, config, state){
-            var response = config.boundTo;
+            var boundTo = config.boundTo;
```

Comment:
can't we use const here?

Reply:
Th ci seems to be broken as well. The is unknown for me. Probably @wazelin can help on this.